File: cmd_to_email.py
import smtplib
from email.mime.text import MIMEText
from email.mime.multipart import MIMEMultipart

EMAIL_PENGIRIM = ""
SANDI_APLIKASI_GOOGLE = "" 
EMAIL_PENERIMA = ""

def kirim_email(subjek, isi_pesan):
    msg = MIMEMultipart()
    msg['From'] = EMAIL_PENGIRIM
    msg['To'] = EMAIL_PENERIMA
    msg['Subject'] = subjek
    msg.attach(MIMEText(isi_pesan, 'plain'))
    
    try:
        print("Sedang Mengirim Pesan")
        server = smtplib.SMTP('smtp.gmail.com', 587)
        server.starttls()
        server.login(EMAIL_PENGIRIM, SANDI_APLIKASI_GOOGLE)
        server.send_message(msg)
        server.quit()
        print("Email berhasil dikirim.")
    except Exception as e:
        print(f"Gagal mengirim email: {e}")

if __name__ == "__main__":
    print("Kirim Pesan Email")
    try:
        while True:
            pesan_input = input("\nMasukkan pesan: ")
            
            if pesan_input.strip().lower() in ['exit', 'keluar']:
                print("Program dihentikan.")
                break
                
            if not pesan_input.strip():
                print("Pesan tidak boleh kosong.")
                continue
                
            subjek_email = "Pesan dari CMD"
            kirim_email(subjek_email, pesan_input)
    except KeyboardInterrupt:
        print("\nexit")

File: cmd_to_email.js
const nodemailer = require('nodemailer');
const prompt = require('prompt-sync')({sigint: true});

// Konfigurasi Email
const EMAIL_PENGIRIM = "";
const SANDI_APLIKASI_GOOGLE = ""; 
const EMAIL_PENERIMA = "";

// Membuat koneksi ke Gmail
const transporter = nodemailer.createTransport({
    service: 'gmail',
    auth: {
        user: EMAIL_PENGIRIM,
        pass: SANDI_APLIKASI_GOOGLE
    }
});

async function jalankan_program() {
    console.log("Kirim Pesan Email");
    
    while (true) {
        console.log("");
        const pesan = prompt('Masukkan pesan: ');
        
        if (pesan === null) {
            console.log("Program dihentikan");
            break;
        }
        
        if (pesan.toLowerCase() === 'exit' || pesan.toLowerCase() === 'keluar') {
            console.log("Program dihentikan");
            break;
        }
        
        if (pesan.trim() === '') {
            console.log("Pesan tidak boleh kosong");
            continue;
        }
        
        // Kirim email
        try {
            console.log("Sedang Mengirim Pesan");
            await transporter.sendMail({
                from: EMAIL_PENGIRIM,
                to: EMAIL_PENERIMA,
                subject: "Pesan dari CMD",
                text: pesan
            });
            console.log("Email berhasil dikirim.");
        } catch (error) {
            console.log(`Gagal mengirim email: ${error.message}`);
        }
    }
}

jalankan_program();